### ISSUE TITLE
[Enhancement] Buttons UI update

### ISSUE THREAD

**Issue by bhansa** [pythonindia/inpycon2018#102]
Currently, the UI of buttons is looking dull with a gray border, all the CTA's should be in either in Flat UI or classic UI design with better color contrast and shadow elevation from the body.

I will share the preview soon.

**Comment by ananyo2012** [comment]
Hi Bharat, there shouldn't be a grey border. I guess this is a caching
issue. Can you try hard reloading or opening in a incognito window? If it
still doesn't work can you share the screenshot of the bug?

**Comment by bhansa** [comment]
you're right @ananyo2012, now I see the different design, previously they were looking as disabled. Let's see if we can still change something there otherwise I'll close this issue.

![image](https://user-images.githubusercontent.com/9723884/40663284-f8cf4552-6375-11e8-8bcf-998c29b53ae1.png)

**Comment by ananyo2012** [comment]
@bhansa Do you want to add something to this or we can close this ?

**Comment by bhansa** [comment]
sorry @ananyo2012 I got busy, will share the preview soon.

**Comment by ananyo2012** [comment]
@bhansa Any update on this?
